data/data/aws/vpc/vpc-private.tf

```diff
@@ -1,5 +1,6 @@
 resource "aws_route_table" "private_routes" {
-  count  = length(var.availability_zones)
+  count = var.private_subnets ? 0 : length(var.availability_zones)
```

Comment:
what if the count is zero ?

Reply:
> what if the count is zero ?

https://github.com/openshift/installer/pull/2438#discussion_r330780170